### PR TITLE
Adds Sentry for React Native

### LINKED ISSUE #511: Install Sentry at RN level
SDK: https://github.com/getsentry/react-native-sentry

### PR DESCRIPTION
**Pull request by alloy**
Currently need to use this until it’s merged and released https://github.com/getsentry/react-native-sentry/pull/50. It’s important that we don’t forget to configure Eigen to use this fork as well!

Closes #511

### CODE REVIEW

File: Pod/Classes/Core/AREmission.m [hunk 1]
```diff
@@ -7,11 +7,13 @@
 
 #import <React/RCTBridge.h>
 #import <React/RCTBridgeModule.h>
+#import <SentryReactNative/RNSentry.h>
 
 
 @interface AREmissionConfiguration : NSObject <RCTBridgeModule>
 @property (nonatomic, strong, readwrite) NSString *userID;
 @property (nonatomic, strong, readwrite) NSString *authenticationToken;
+@property (nonatomic, strong, readwrite) NSString *sentryDSN;
```
Comment (orta) [inline Pod/Classes/Core/AREmission.m]: Minor nit: these strings should probably be `copy`

File: Example/Emission/UnroutedViewController.m [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #import "UnroutedViewController.h"
-#import <Artsy+UIFonts/UIFont+ArtsyFonts.h>
+#import <Artsy_UIFonts/UIFont+ArtsyFonts.h>
 #import <FLKAutoLayout/FLKAutoLayout.h>
```
Comment (orta) [inline Example/Emission/UnroutedViewController.m]: Hah

We could move to `@import Artsy_UIFonts` too

File: index.ios.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
-import Containers from './src/lib/containers'
-import Components from './src/lib/components'
-import Routes from './src/lib/relay/routes'
+// First things first.
+import "./src/lib/error_reporting"
```
Comment (orta) [inline index.ios.js]: I can't find this file?

screen shot 2017-05-12 at 09 02 23

**Comment by alloy** [comment]
Ooooh, great catch @orta

**Comment by alloy** [comment]
@orta Added

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Missing Test Files:

- `src/lib/__tests__/error_reporting-tests.ts`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by alloy** [comment]
Alright, gonna go for the merge

**Pull request merged**